crates/uv-resolver/src/requires_python.rs

```diff
@@ -460,6 +541,65 @@ impl RequiresPythonBound {
             Bound::Unbounded => Bound::Unbounded,
         })
     }
+
+    fn to_marker_tree(&self, greater: bool) -> MarkerTree {
```

Comment:
Can you document `greater`? Or make it an enum to self-document a bit?

Reply:
Thankfully I just got rid of this entirely as your `markers()` implementation (with that bug fix) now works.

(I'm also not a huge fan of boolean parameters either, but tend to abide them for unexported helpers that are used once or twice in lieu of adding another type. But I don't feel too strongly about it.)